static/templates/test/and.hbs

```diff
@@ -0,0 +1,2 @@
+{{#if (and)}}<p>empty and</p>{{/if}}
+{{#if (and true last)}}<p>last and</p>{{/if}}
```

Comment:
An ideal test for this would include an example where the and combines false with something, right?